src/imputation/imputation_main.py

```diff
@@ -50,6 +50,15 @@ def run_imputation(
     Returns:
         pd.DataFrame: dataframe with the imputed columns updated
     """
+    # add osmotherly & key businesses columns to PNP data
+    if config["survey"]["survey_type"] == "PNP":
+        df = hlp.identify_key_business(df)
+        df = hlp.identify_osmotherly_businesses(df)
+        df = hlp.add_area_column(df)
```

Comment:
I suggest this all moves to the mapping module.

Reply:
Done :)